### DISCUSSION TITLE
Looking at screens

### DISCUSSION THREAD

**Discussion by michaelneale** [aaif-goose/goose#25] [Show and tell]
there is a screen toolkit you can enable for goose (currently just with openai provider - but can in future work with others) that lets you look at the screen. 

I thought I would show this doing something a bit... trivial, which is it looking at my second screen, and trying, from visuals only, to match styles in the browser on the right with the browser on the left (and told it to auto refresh). Not sure exactly what I would do this for, but thought it was fun. 

One practical use I have done is with a dart mobile app, have it move a button around (I don't know dart) and then it watches the simulator. Could be used I guess to help you get closer to a design which is not in code?

![imageCopy](https://github.com/user-attachments/assets/89303964-8762-465d-b0af-311ef553e58a)

**Comment by blackgirlbytes** [comment]
Closing this discussion since Goose has evolved so much since.
